Assets/Samples/Volume Framework/Scripts/ToonSharpenVolumeComponent.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+using UnityEngine.Rendering;
+using static DELTation.ToonRP.PostProcessing.BuiltIn.ToonSharpenSettingsTest;
+
+[VolumeComponentMenu("ToonRP/Toon Sharpen")]
+public class ToonSharpenVolumeComponent : VolumeComponent
+{
+    public ClampedFloatParameter Amount = new ClampedFloatParameter(0f, 0f, 10f);
+    public EnumParameter<PassOrder> Order = new EnumParameter<PassOrder>(PassOrder.PreUpscale);
```

Comment:
I'm not sure whether the Order should be overridable, it may causes difficulties since changing it properly would require reordering passes.